applications/nrf5340_audio/README.rst

```diff
@@ -780,12 +780,17 @@ Selecting the BIS mode
 ======================
 
 The CIS mode is the default operating mode for the application.
-You can switch to the BIS mode by adding the ``CONFIG_TRANSPORT_BIS`` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
-To build the BIS gateway with the alternative name, given by ``CONFIG_BT_DEVICE_NAME_ALT``, add the option ``CONFIG_BT_USE_DEVICE_NAME_ALT`` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
+You can switch to the BIS mode by adding the :kconfig:option:`CONFIG_TRANSPORT_BIS` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
 
-.. note::
-   The gateway firmware must be built twice to give two separate gateway devices two separate names (``CONFIG_BT_DEVICE_NAME`` and ``CONFIG_BT_DEVICE_NAME_ALT``).
-   Program the two firmware versions to two separate gateway devices.
+Enabling the BIS mode with two gateways
+---------------------------------------
+
+In addition to the standard BIS mode with one gateway, you can also add a second gateway device that the BIS headsets can receive audio stream from.
+To configure the second gateway, add both the :kconfig:option:`CONFIG_TRANSPORT_BIS` and the :kconfig:option:`CONFIG_BT_USE_DEVICE_NAME_ALT` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and to the :file:`prj_release.conf` file for the release version.
+You can provide an alternative name to the second gateway using the :kconfig:option:`CONFIG_BT_DEVICE_NAME_ALT` or use the default alternative name.
+
+You build the BIS gateways using the normal procedures from :ref:`nrf53_audio_app_building`, but the gateway firmware must be built twice to give two separate gateway devices two separate names (``CONFIG_BT_DEVICE_NAME`` and ``CONFIG_BT_DEVICE_NAME_ALT``).
```

Comment:
This is a confusing sentence. To be clear: If you want an additional gateway that you can sync to, build with CONFIG_BT_USE_DEVICE_NAME_ALT=y and flash to a second gateway. 

Now it seems that you can just build the same configuration twice and it's magically two different gateways

Reply:
Hmmm the next sentence is "Remember to program the two firmware versions to two separate gateway devices.", which to me makes it pretty clear that two different gateway devices are needed

Reply:
Checking with @erikrobstad

Reply:
Latest changes look better, resolved

Reply:
Ah, you probably meant that it is not mentioned in this sentence that CONFIG_BT_USE_DEVICE_NAME_ALT=y has to be added for the second gateway